x/src/test.rs

```diff
@@ -17,16 +23,66 @@ pub struct Args {
     #[structopt(long, short)]
     /// Only run unit tests
     unit: bool,
+
+    #[structopt(long)]
+    /// Do not fast fail the run if tests (or test executables) fail.
+    no_fail_fast: bool,
+    #[structopt(long)]
+    /// Do not run tests, only compile the test executables.
+    no_run: bool,
+
+    #[structopt(long, parse(from_os_str))]
+    /// Setting an hmtlcovdir will result in grcov being run for the workspace, or crate the
+    /// command is executed from.   A clean will be performed before running the tests.
```

Comment:
Maybe something short like:

```suggestion
    /// Directory to output HTML coverage report to (using grcov)
```

Reply:
done.